owncloudApp/src/main/java/com/owncloud/android/presentation/ui/authentication/LoginActivity.kt

```diff
@@ -403,11 +476,11 @@ class LoginActivity : AppCompatActivity(), SslUntrustedCertDialog.OnSslUntrusted
         TokenResponseCallback { tokenResponse: TokenResponse?, authorizationException: AuthorizationException? ->
             if (tokenResponse?.accessToken != null && tokenResponse.refreshToken != null) {
                 authenticationViewModel.loginOAuth(
-                    tokenResponse.additionalParameters[AccountUtils.Constants.KEY_USER_ID]!!,
+                    tokenResponse.additionalParameters[AccountUtils.Constants.KEY_USER_ID]?.let { it } ?: "",
```

Comment:
You can remove that `let` and use simply `?:`, can't you?

Reply:
Sure 🤦